README.md

```diff
@@ -27,6 +27,7 @@ Options:
   -k, --keep                 reuse docker container (prevents deleting)
   -j, --jtreg                run jtreg after building
   -S, --ssh                  use ssh when cloning git
+  --variant <name>           specify a build variant name, e.g. openj9
```

Comment:
minor niggle: we usually have a short and long param option, if `-v` or `-V` is available would be good to use